src/PostgREST/Parsers.hs

```diff
@@ -0,0 +1,175 @@
+module PostgREST.Parsers
+( parseGetRequest
+)
+where
+
+import           Control.Applicative hiding ((<$>))
+import           Data.Functor ((<$>))
+import           Data.Traversable (traverse)
```

Comment:
Oh good call. Maybe a comment to that effect by those lines would be helpful. Or a [CPP macro](http://www.edsko.net/2014/09/13/haskell-cpp-macros/) which only includes the extra imports when compiling in GHC <7.10. Maybe that's overkill though. :) Up to you.